core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -2039,6 +2151,32 @@ public void removeFog(String... fogNameSpaces) {
         return Set.copyOf(this.appliedFog);
     }
 
+    @Override
+    public void lockInputs(boolean camera, boolean movement) {
+        UpdateClientInputLocksPacket packet = new UpdateClientInputLocksPacket();
+        final int cameraOffset = 1 << 1;
+        final int movementOffset = 1 << 2;
+
+        int result = 0;
+        result |= (camera ? cameraOffset : 0);
+        result |= (movement ? movementOffset : 0);
```

Comment:
i would use 2 if statements, rather than using bitwise OR with `0` as a way to do nothing